raiden/tests/unit/storage/migrations/test_v20_to_v21.py

```diff
@@ -0,0 +1,123 @@
+import json
+from datetime import datetime
+from pathlib import Path
+from unittest.mock import patch
+
+from raiden.storage.migrations.v20_to_v21 import upgrade_v20_to_v21
+from raiden.storage.sqlite import SQLiteStorage
+from raiden.utils.upgrades import UpgradeManager, UpgradeRecord
+
+
+def setup_storage(db_path):
+    storage = SQLiteStorage(str(db_path))
+
+    state_changes_file = Path(__file__).parent / 'data/v20_statechanges.json'
+    state_changes_data = json.loads(state_changes_file.read_text())
+    for state_change_record in state_changes_data:
+        storage.write_state_change(
+            state_change=json.dumps(state_change_record[1]),
+            log_time=datetime.utcnow().isoformat(timespec='milliseconds'),
+        )
+
+    chain_state_data = Path(__file__).parent / 'data/v20_chainstate.json'
```

Comment:
we should probably not do this `'data/v20_chainstate.json'`, this is not a huge problem but it does not handle cross os compatibility. We should either use:

```python
Path().parent / 'data' / 'v20_chainstate.json'
```

or 

```python
os.path.join(os.path.dirname(__file__), 'data', 'v20_chainstate.json')
```